src/frameworks/docs/flask.md

```diff
@@ -0,0 +1,60 @@
+Project: /docs/hosting/_project.yaml
+Book: /docs/_book.yaml
+page_type: guide
+
+{% include "_shared/apis/console/_local_variables.html" %}
+{% include "_local_variables.html" %}
+{% include "docs/hosting/_local_variables.html" %}
+
+<link rel="stylesheet" type="text/css" href="/styles/docs.css" />
+
+# Integrate Flask
+
+With the Firebase framework-aware {{cli}}, you can deploy your Flask application
+to Firebase and serve dynamic content to your users.
+
+Note: Flask support in Framework-aware {{hosting}} is coming soon.
```

Comment:
The distinction may not be very clear at this point.  Note that we basically conflate framework-aware CLI and framework-aware Hosting in pages like https://firebase.google.com/docs/hosting/frameworks/angular

Reply:
I can understand both but arguably the implementation *ATM* is that the CLI is framework aware and doing all the heavy lifting. IMO we should make it consistent.

Reply:
Sounds true!  But right here, in line, what is the difference/distinction between

"With the Firebase framework-aware {{cli}}, you can deploy your Flask application to Firebase and serve . . ."

and

"Flask support in Framework-aware {{hosting}}"?

It sounds to me like we are saying "here's a thing, and the same thing is coming soon."  If there's a meaningful difference between what you get after you finish the steps in this guide and what you'll get coming soon, then I think we need more words to clarify the difference between those two things.